CoffeeChat/Controller/SocialMediaViewController.swift

```diff
@@ -0,0 +1,202 @@
+        skipButton.setTitle("Skip", for: .normal)
+        skipButton.setTitleColor(.greenGray, for: .normal)
+        skipButton.backgroundColor = .none
+        skipButton.addTarget(self, action: #selector(skipButtonPressed), for: .touchUpInside)
+        view.addSubview(skipButton)
+
+        setupConstraints()
+    }
+
+    private func setupConstraints() {
+
+        backButton.snp.makeConstraints { make in
+            make.top.equalTo(titleLabel).offset(6)
+            make.size.equalTo(Constants.Onboarding.backButtonSize)
+            make.leading.equalToSuperview().offset(24)
+        }
+
+        titleLabel.snp.makeConstraints { make in
+            make.centerX.equalToSuperview()
+            make.size.equalTo(CGSize(width: 318, height: 61))
+            make.top.equalTo(view.safeAreaLayoutGuide).offset(Constants.Onboarding.titleLabelPadding)
+        }
+
+        subtitleLabel.snp.makeConstraints { make in
+            make.top.equalTo(titleLabel.snp.bottom).offset(12)
+            make.centerX.equalToSuperview()
+            make.height.equalTo(20)
+        }
+
+        disclaimerLabel.snp.makeConstraints { make in
+            make.bottom.equalTo(nextButton.snp.top).offset(-16)
+            make.leading.trailing.equalToSuperview().inset(63)
+        }
+
+        instagramTextField.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview().inset(40)
+            make.height.equalTo(49)
+            make.top.equalTo(subtitleLabel.snp.bottom).offset(62)
+        }
+
+        facebookTextField.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview().inset(40)
+            make.height.equalTo(49)
+            make.top.equalTo(instagramTextField.snp.bottom).offset(16)
+        }
+
+        nextButton.snp.makeConstraints { make in
+            make.size.equalTo(Constants.Onboarding.mainButtonSize)
+            make.centerX.equalToSuperview()
+            make.bottom.equalTo(view.safeAreaLayoutGuide).inset(Constants.Onboarding.nextBottomPadding)
+        }
+
+        skipButton.snp.makeConstraints { make in
+            make.size.equalTo(CGSize(width: 86, height: 20))
+            make.centerX.equalToSuperview()
+            make.bottom.equalTo(view.safeAreaLayoutGuide).inset(Constants.Onboarding.skipBottomPadding)
+        }
+
+    }
+
+
+    // MARK: - Next and Previous Buttons
+    private func updateNext() {
+
+        guard let instagramHandle = instagramTextField.text, let facebookHandle = facebookTextField.text else { return }
+        let isSocialMediaEntered = instagramHandle.trimmingCharacters(in: .whitespaces) != "" && facebookHandle.trimmingCharacters(in: .whitespaces) != ""
+        nextButton.isEnabled = isSocialMediaEntered
+        nextButton.backgroundColor = nextButton.isEnabled ? .backgroundOrange : .inactiveGreen
+        skipButton.isEnabled = !nextButton.isEnabled
+        let skipButtonColor: UIColor = skipButton.isEnabled ? .greenGray : .inactiveGreen
```

Comment:
Might you be able to skip the explicit type declaration because the compiler can figure it out from `skipButton.setTitleColor`?